src/encoded/static/libs/ui/panel.js

```diff
@@ -272,14 +272,16 @@ class TabItem extends React.Component {
     }
 
     clickHandler() {
-        this.props.handleClick(this.props.tab);
+        if (!(this.props.children && this.props.children.props && this.props.children.props.className && this.props.children.props.className.includes('disabled'))) {
```

Comment:
I recommend the condition be put in a method as seeing a long check twice is a code smell.

Reply:
Excellent point.